odc/stats/plugins/l34_utils/lc_level3.py

```diff
@@ -46,7 +46,7 @@ def lc_level3(xx: xr.Dataset):
     # Add intertidal as water
     res = expr_eval(
         "where((a==223)|(a==221), 220, b)",
-        {"a": xx.classes_l3_l4.data, "b": res},
+        {"a": xx.level_3_4.data, "b": res},
         name="mark_urban",
         dtype="uint8",
     )
```

Comment:
need to sum aquatic classes as well

Reply:
Added a statement for combining 124 and 125.